Products/CMFPlone/browser/favicon.py

```diff
@@ -1,4 +1,4 @@
-from os.path import dirname
+from os.path import dirname, join
```

Comment:
one import per line, alphabetical sorted please

Reply:
Done.  Please stamp approve when you see fit.